### DISCUSSION TITLE
Edit recipient addresses in airstream campaigns

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/v2-periphery#270] [Ideas]
Feature request to allow an airstream campaign creator to edit the recipient addresses:

<img width="400" alt="SCR-20240131-ryqh" src="https://github.com/sablier-labs/v2-periphery/assets/8782666/3f9da2ac-1f17-49df-ac39-ca64889b41d8">

I don't think this is possible in the Merkle API (cc @gavriliumircea) but I've posted the discussion just in case.

**Comment by gavriliumircea** [comment]
That is definitely not possible in the Merkle API. Any change to the recipient change the whole merkle tree and thus all the proofs become unusable.
